data/campaigns/Secrets_of_the_Ancients/units/Orcish_Shaman.cfg

```diff
@@ -1,101 +1,16 @@
 #textdomain wesnoth-sota
 [unit_type]
-    id=Orcish Shaman
+    id=SotA Orcish Shaman
```

Comment:
Why is it necessary to use a copy of the unit? Can we not like change the unit to Orcish Adept?

Reply:
I think this stemmed from spotting some differences between the SotA shaman and the SotBE shaman, but I don't remember clearly.

Reply:
It should be standardized. If necessary we could use an [object] to change the damage.

Reply:
SotA one removed in 402215a